packages/manager/modules/support/src/tickets/new-ticket/issues-form/issues-form.html

```diff
@@ -1,108 +1,111 @@
-<form novalidate
-      name="supportNewIssuesForm">
+<form
+    novalidate
+    name="supportNewIssuesForm">
 
     <div class="text-center">
-        <oui-spinner data-ng-if="$ctrl.isLoading"
-                     data-size="l">
-        </oui-spinner>
+        <oui-spinner
+            data-ng-if="$ctrl.bindings.category.isLoading"
+            data-size="l"></oui-spinner>
     </div>
 
-    <div data-ng-if="!$ctrl.isLoading">
-        <oui-field data-label="{{:: 'ovhManagerSupport_new_category' | translate }}"
-                   data-size="xl">
-            <oui-select name="category"
-                        data-model="$ctrl.category"
-                        data-items="$ctrl.categories"
-                        data-match="label"
-                        data-on-change="$ctrl.onCategoryChange()">
-            </oui-select>
-        </oui-field>
-        <oui-field data-label="{{:: 'ovhManagerSupport_new_serviceType' | translate }}"
-                   data-ng-if="$ctrl.serviceType || ($ctrl.category && $ctrl.category.id !== $ctrl.CATEGORY_ACCOUNT)"
-                   data-size="xl">
-            <oui-select name="serviceType"
-                        data-model="$ctrl.serviceType"
-                        data-items="$ctrl.serviceTypes"
-                        data-match="label"
-                        data-on-change="$ctrl.onServiceTypeChange()"
-                        data-searchable>
-            </oui-select>
+    <oui-field
+        data-ng-if="!$ctrl.bindings.category.isLoading"
+        data-label="{{:: 'ovhManagerSupport_new_category' | translate }}"
+        data-size="xl">
+        <oui-select
+            data-model="$ctrl.bindings.category.value"
+            data-items="$ctrl.bindings.category.values"
+            data-match="label"
+            data-on-change="$ctrl.onCategoryChange()"
+            name="category"></oui-select>
```

Comment:
```suggestion
            data-name="category"></oui-select>
```
As `name` is a possible property of the component

Reply:
Good point 👍